vnext/just-task.js

```diff
@@ -33,14 +37,21 @@ task('ts', () => {
       sourceRoot: path.relative(libPath, srcPath),
     }),
     target: 'es5',
-    outDir: 'lib',
     module: 'commonjs',
   });
 });
 task('clean', () => {
   return cleanTask(
-    ['lib', 'temp', 'dist', 'coverage'].map(p => path.join(process.cwd(), p)),
+    ['Libraries', 'RNTester', 'lib'].map(p => path.join(process.cwd(), p)),
```

Comment:
>RNTester [](start = 19, length = 8)

The getting started guide ([https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/GettingStarted.md)) will need updating to point folks at the new location for RNTester.